### PR TITLE
KUZBO-116 Enriched user list as per mockup

### PR DESCRIPTION
**Pull request by xbill82**
The list items in the user list section have been enriched according to the [Balsamiq Mockup](https://kuzzleio.mybalsamiq.com/projects/kuzzle-bo-ux/%5Bsecurity%5D%20Users%20%28collapsed%20filters%29) we discussed.

~~:warning: This branch also merges all the commits from @stafyniaksacha #21 in order to have the new multi-profile fixtures.~~

TODO
- [x] Tests

### CODE REVIEW

**Comment by jenow** [comment]
+1

File: src/components/Security/Users/UserItem.vue [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+    <div class="right actions">
+      <a class="action fa fa-pencil"></a>
+      <dropdown :id="user.id">
+        <li><a @click="deleteUser(user.id)">Delete</a></li>
+      </dropdown>
+    </div>
+
+    <div class="item-content">
+      <pre>{{itemContent}}</pre><div class="profile-list">
```
Comment (xbill82) [inline src/components/Security/Users/UserItem.vue]: By the way, we had a proper JSON View component in the old BO? Might be better than a `pre`

Comment (jenow) [inline src/components/Security/Users/UserItem.vue]: it is the same but in the old bo it's wrap into an angular module

Comment (xbill82) [inline src/components/Security/Users/UserItem.vue]: Ok, I'll use your directive once #19 is merged

Comment (xbill82) [inline src/components/Security/Users/UserItem.vue]: Ok, done.

**Comment by stafyniaksacha** [comment]
+1

**Pull request merged**